working-with-your-data/working-with-your-own-data.md

```diff
@@ -1,5 +1,6 @@
 ---
 title: Working with your own data
+nav_title: Work with your Data
```

Comment:
I'm trying to come up with something a bit shorter here, but having trouble. 
"Data management"?
"Using your own data"? (that's the same length.)
"Managing Data"?

Reply:
I haven't looked at this locally to see how it looks, but as far as indicating what the content is I feel that "Work with your data" or "Using your own data" is about right. We could do "Your Data" but that seems a little weird?

Reply:
Okay, How about this? 
```suggestion
nav_title: Using Your Data
```
If not, we should switch to a (modified) title case for better emphasis.
```suggestion
nav_title: Work with Your Data
```

Reply:
I think I prefer "Using Your Data." Looks neater to me for some reason.

Reply:
Okay, then you can just click the "Commit suggestion" button to make that change!